.github/workflows/release.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
     #   4. the title prefix 'chore: Release '.
     if: >
       github.event.pull_request.merged &&
-      github.ref == 'main' &&
+      github.ref == 'refs/heads/master' &&
```

Comment:
Should this be `refs/heads/main` instead? (looks like this repo uses `main` instead of `master`)

Reply:
oh thank you!